lib/node-onvif.js

```diff
@@ -52,9 +52,9 @@ Onvif.prototype.startDiscovery = function(callback) {
 };
 
 /* ------------------------------------------------------------------
-* Method: startProbe([callback])
+* Method: startProbe([address, callback])
 * ---------------------------------------------------------------- */
-Onvif.prototype.startProbe = function(callback) {
+Onvif.prototype.startProbe = function(address, callback) {
```

Comment:
While traditionally the callback is the last argument, I think you should make it the second argument. If you look at `_execCallback` it's done in the same style - https://github.com/LGSInnovations/node-onvif/pull/1/files#diff-630a7d91ebc54e8fae0380339b58c5d4R164 - Also, you would do the same in `_sendProbe` and can eliminate the `typeof address === function` logic.
```suggestion
Onvif.prototype.startProbe = function(callback, address) {
```